events/registration/views.py

```diff
@@ -0,0 +1,64 @@
+import json
+
+from django.core.exceptions import PermissionDenied
+from django.views import View
+from django.contrib.auth.models import User
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.http import JsonResponse
+from django.shortcuts import redirect, reverse
+
+from .models import User, RegistrationConfirm
+from .forms import RegistrationForm
+from events import settings
+
+
+CONFIRM_LINK = settings.HOST_NAME + '/api/v1/reg/confirm/'
+
+
+class EmailSender(object):
+    @staticmethod
+    def send_registration_confirm(user):
+        hash_code = RegistrationConfirm.create_confirm(user)
+        subject = 'Confirm registration'
+        message = render_to_string('email.txt', {'link': CONFIRM_LINK + hash_code, 'name': user.first_name})
+        return send_mail(subject, message, settings.EMAIL_HOST_USER, [user.email]) == 1
+
+
+class RegistrationView(View):
+    def get(self, request):
+        data = {'first_name': '', 'last_name': '', 'email': '', 'password': '', 'birth_date': ''}
+        return JsonResponse(data, status=200)
+
+    def post(self, request):
+        body_unicode = request.body.decode('utf-8')
+        data = json.loads(body_unicode)
+        registration_form = RegistrationForm(data)
+        if registration_form.is_valid():
+            user = User()
```

Comment:
Please replace this with 
```
User.objects.create(
username = data.get('email'),
first_name = data.get('first_name'),
last_name = data.get('last_name'),
email = data.get('email'),
is_active = False,
password = set_password(data.get('password'))
)
```
The reason is that save() is dangerous and we need to avoid of it's usage.